pynamodb/models.py

```diff
@@ -417,31 +417,43 @@ def delete(self, condition: Optional[Condition] = None, settings: OperationSetti
 
         return self._get_connection().delete_item(hk_value, range_key=rk_value, condition=condition, settings=settings)
 
-    def update(self, actions: List[Action], condition: Optional[Condition] = None, settings: OperationSettings = OperationSettings.default) -> Any:
+    def update(
+        self,
+        actions: List[Action],
+        condition: Optional[Condition] = None,
+        settings: OperationSettings = OperationSettings.default,
+        read_back: str = ALL_NEW,
```

Comment:
I wonder if we should make it an enum. This library isn't awfully modern, but maybe there's an opportunity to change that (especially as we're establishing a new API here).

Reply:
I'd say an enum is a better choice, too. Let's start to use strong typing whenever possible.